doc/ir-semantics.text

```diff
@@ -108,18 +118,30 @@ Update (partial) function, returns a S:
        = step CC[v] (P, T, H, E*, C*)
     when v := eval P H E e
 
+  step   CC[call f (v, ...)]
+         (P, T, H, E*, C*)
+       = (P, T, H, E* E', C* C)
+    when P(f)  = Function (x, ...)
+     and E'   := (x ↦ v), ...
+     and C    := (f, active, start)
+
+  step   (return v)
+         (P, T, H, E* E', C* C C')
+       = step cs (P, T, H, E*, C* C)
+    when cs := P(C)[call(...)/v]
```

Comment:
okok, I am just saying, that so far it was possible to have no expressions in continuations. If you want me to fix this, then we will have expressions in continuations.